### DISCUSSION TITLE
[Question]: List of platform which supports for deployment

### DISCUSSION THREAD

**Discussion by kination** [spaship/spaship#1360] [Q&A] [answered]
Hello, have some question about this project(sorry I couldn't find the document)

Is this for deploying SPA to the platform? Is it also support deployment to AWS(such as S3?), or GCP?

**Comment by sayak-sarkar** [comment, marked as answer]
@djKooks Yes, SPAship is a tool that manages SPA deployments to appropriately configured SPAship platforms. We haven't explored barebones AWS or GCP but we have run SPAship on baremeta systems, VMs and on Openshift instances deployed on AWS. Ideally you should be able to configure a SPAship instance to run on any cloud platform of your choice and deploy SPAs to it.

Hope that helps! :)

**Comment by kination** [comment]
@sayak-sarkar thanks for comment, but still not quite sure how to use this for deployment to cloud system.
Is there some user guide?
